### PR TITLE
haproxy 1.6.0

### PR DESCRIPTION
**Pull request by gianarb**

### CODE REVIEW

File: haproxy/1.6.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:latest
+MAINTAINER Gianluca Arbezzano <gianarb92@gmail.com>
+
+RUN apt-get update
+RUN apt-get install -y wget build-essential libpcre3-dev libssl-dev
+RUN wget http://www.haproxy.org/download/1.6/src/haproxy-1.6.0.tar.gz
+RUN tar -xzvf haproxy-1.6.0.tar.gz
+WORKDIR ./haproxy-1.6.0
+
```
Comment (wdalmut) [inline haproxy/1.6.0/Dockerfile]: Effectively with: 

```
RUN touch doc/haproxy-en.txt
RUN touch doc/haproxy-fr.txt
```

it compiles correctly

**Comment by wdalmut** [comment]
The `make` is working the problem is with the `install` section that need those files...

**Comment by wdalmut** [comment]
Or you can skip the whole install and use the `install-bin`

```Dockerfile
FROM ubuntu:latest
MAINTAINER Gianluca Arbezzano <gianarb92@gmail.com>

RUN apt-get update
RUN apt-get install -y wget build-essential libpcre3-dev libssl-dev
RUN wget http://www.haproxy.org/download/1.6/src/haproxy-1.6.0.tar.gz
RUN tar -xzvf haproxy-1.6.0.tar.gz
WORKDIR ./haproxy-1.6.0

RUN make TARGET=linux26 USE_PCRE=1 CPU=generic USE_OPENSSL=1 USE_LIBCRYPT=1
RUN make install-bin

CMD ["haproxy", "-v"]
```

File: haproxy/1.6.0/Dockerfile (hunk 1 above)
Comment (gianarb) [inline haproxy/1.6.0/Dockerfile]: It sounds correct but it is very ugly :P Thanks

**Comment by gianarb** [comment]
make install-bin.. Thanks.. I don't understand why this choice.. :) but ok :D

**Comment by wdalmut** [comment]
@gianarb i think that this error not something scheduled by haproxy. If you take a look in their makefile at https://github.com/haproxy/haproxy/blob/master/Makefile#L804 there is a loop that is seeded with a wrong list maybe...

**Comment by wdalmut** [comment]
@gianarb the command

```sh
make install
```

Is composed by 3 tasks:

```make
install: install-bin install-man install-doc
```

The last step return with not 0 exit code

**Comment by gianarb** [comment]
Yes I understand the error.. I don't know why this failure did not been reported from nobody :)

**Comment by gianarb** [comment]
Thanks @wdalmut

**Comment by wdalmut** [comment]
@gianarb maybe we can report this issue to haproxy...

**Comment by gianarb** [comment]
I'm trying to ping about this problem on IRC but I didn't receive
response.. :)

I try to write an issue..

2015-10-18 14:58 GMT+02:00 Walter Dal Mut <notifications@github.com>:

> @gianarb <https://github.com/gianarb> maybe we can report this issue to
> haproxy...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gianarb/dockerfile/pull/3#issuecomment-149009784>.
>



-- 
Gianluca Arbezzano
www.gianarb.it

**Pull request merged**